pantheon-systems/drops-8
# Update to Drupal 8.0-dev-2015-11-17.

Drupal 8.0.0-rc4 was not deployed due to a cosmetic bug.  This PR contains a dev release of Drupal 8 from just a couple days prior to the anticipated Drupal 8.0.0 stable release.

Instructions to test:

- Create a new Drupal 8 site on Pantheon; start with Drupal 8.0.0-rc3.
- When site creation is finished, visit dashboard.
- Switch to "git" mode.
- Clone your site locally.
- Apply the files from this PR on top of your local checkout.
  - git remote add drops-8 git@github.com:pantheon-systems/drops-8.git
  - git fetch drops-8
  - git merge drops-8/update-8.0-dev-2015-11-17
- Push your files back up to Pantheon.
- Switch back to sftp mode.
- Visit your site. Step through the installation process. Look for any problems.

Note that the "minimal" installation profile is still broken on Pantheon (c.f. #96), so please use the standard installation profile to test.
